### ISSUE TITLE
updateUserMemberships - Wrong documentation for parameters

### ISSUE THREAD

**Issue by bloigge** [Esri/arcgis-rest-js#1124]
### Description

The documented parameters for the `updateUserMemberships` method are wrong: 

```js
import { updateUserMemberships } from "@esri/arcgis-rest-portal";

updateUserMemberships({
  id: groupId,
  admins: ["username3"],
  authentication
})
.then(response);
```

should be:

```js
import { updateUserMemberships } from "@esri/arcgis-rest-portal";

updateUserMemberships({
  id: groupId,
  users: ["username3"],
  newMemberType: "admin",
  authentication
})
.then(response);
```

### Link to page

[https://developers.arcgis.com/arcgis-rest-js/api-reference/arcgis-rest-portal/updateUserMemberships](https://developers.arcgis.com/arcgis-rest-js/api-reference/arcgis-rest-portal/updateUserMemberships)

**Comment by gavinr-maps** [comment]
Hi @bloigge, thank you for logging this. That documentation is automatically generated from the code comments in this repo. It looks like the snippet in question is here: 

https://github.com/Esri/arcgis-rest-js/blob/c8198f4b65694018302f10e0e5f30277a05998d7/packages/arcgis-rest-portal/src/groups/update-user-membership.ts#L32-L41

Would you be able to submit a pull request with the update? Thank you!

**Comment by bloigge** [comment]
Hi @gavinr-maps ,
For any open-source project I would do that right away. But as we are customers of ESRI Enterprise, I think it is the responsibility of ESRI to keep the documentation clean for their customers!

**Comment by alhridoy** [comment]
Hi @gavinr-maps  I can work on that. Please assign me for that.

**Comment by gavinr-maps** [comment]
Hi @alhridoy, thanks for the note. No need to be assigned - just submit a PR!

**Comment by alhridoy** [comment]
> Hi @alhridoy, thanks for the note. No need to be assigned - just submit a PR!

Hi @gavinr-maps I created the PR. Please check this out. Thanks

**Comment by gavinr-maps** [comment]
fixed in #1132. thanks @alhridoy!
